src/components/Navbar.js

```diff
@@ -225,13 +230,15 @@ const NavBar = ({ auth, history, location }) => {
                 : Logo
             }
             alt="Find Ur City Logo"
+            
+            
           />
         </Link>
       </H2>
       {token ? (
         <UL>
           <Li className={classes.avatarBox}>
-            <img src={avatar} alt="avatar" />
+            <img src={profileImage} alt="avatar" style={{ height: '60px', width: '60px', borderRadius: '50%', border: '3px solid #2ec7bf ' }} />
```

Comment:
This looks great Conary. I love the idea of having the profile picture as an avatar. It doesn't look great tho when the user did not upload a picture. You can add some if/else logic to display a default image for users without a profile picture.